webrtc.html

```diff
@@ -7476,6 +7476,16 @@ <h2>Methods</h2>
               include this {{RTCRtpTransceiver}} until this method is
               called again. Setting <var>codecs</var> to an empty sequence
               resets codec preferences to any default value.</p>
+              <p class="note">Codecs have their payload types listed
+              under each m= section in the SDP, defining the mapping between
+              payload types and codecs. These payload types are referenced by
+              the m=video or m=audio lines in the order of preference, and
+              codecs that are not negotiated do not apear in this list.
+              A previously negotiated codec that is subsequently removed
+              disappears from the m=video or m=audio line, and while its codec
+              payload type is not to be reused in future offers or answers,
+              its payload type may also be removed from the mapping of payload
+              types in the SDP.</p>
```

Comment:
I think this is consistent with JSEP section 5.2.1:

```
   o  If codec preferences have been set for the associated transceiver,
      media formats MUST be generated in the corresponding order, and
      MUST exclude any codecs not present in the codec preferences.

   o  Unless excluded by the above restrictions, the media formats MUST
      include the mandatory audio/video codecs as specified in
      [RFC7874], Section 3, and [RFC7742], Section 5.
```
Should we have a reference to that section somewhere?

Reply:
Yes, that seems like a good idea (though the reference doesn't cover the "not to be reused" aspect).

Reply:
That's general SDP rules, I think.